docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        return yyyymmdd.cast("Date")
</pre><p>Can you change this to <code>"date"</code>? Technically the parser is case insensitive, but conventionally we've preferred lower case.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Done.</p>